cubano-config/src/main/java/org/concordion/cubano/config/ProxyConfig.java

```diff
@@ -176,10 +176,18 @@ private void setProxyFromEnvironmentVariables() {
                 proxyPassword = System.getenv("HTTP_PROXY_PASS");
             }
 
-            nonProxyHosts = System.getenv("NO_PROXY");
+			nonProxyHosts = System.getenv("NO_PROXY");
```

Comment:
Hi @jimmykemp , can you change from using tabs to spaces and fix it up in these changes please? Otherwise the layout is inconsistent dependent on tab width.

Reply:
@nigelcharman hope that commit is better. Might not have had the formatter defaulting to cubano earlier.